### PR TITLE
#798 fix bad table

### LINKED ISSUE #798: Fix bad table formatting in documentation
A lot of the tables in the Epidata documentation for the COVIDcast endpoint are badly formatted. (They may also be badly formatted in other places in the Epidata documentation, I haven't looked as carefully.)

For example, looking at the HHS data source this is what I see: 

Screen Shot 2022-01-06 at 12 09 13 PM

And when I scroll right, I get this: 

Screen Shot 2022-01-06 at 12 09 06 PM

We should fix this.

### PR DESCRIPTION
**Pull request by LeonLu2**
Fixes #798 

1. reformat the table
2. combine the signal and 7-day signal columns to one column
3. remove the geography and resolution columns and add universal explain below the table to make the description column text more visible

### CODE REVIEW

**Review by krivard — CHANGES_REQUESTED** [review]
Preview:
image

Please change the description text back to left alignment; center alignment is mismatched with the other pages and is difficult to read.

The raw counts and 7dav signals needn't be separated by newlines, and we should switch from ampersands to `and`; see [the CTIS page](https://github.com/cmu-delphi/delphi-epidata/blob/dev/docs/api/covidcast-signals/fb-survey.md) for an example.

**Comment by LeonLu2** [comment]
Sure, the changes are made and up-to-date. Is there any other issues related to the table?

**Comment by krivard** [comment]
Great! Apply same fix to JHU and USAFacts.

**Comment by LeonLu2** [comment]
For the table issue, JHU and USAFacts pages are all fixed.

File: docs/api/covidcast-signals/hhs.md [hunk 1]
```diff
@@ -32,15 +32,16 @@ adult and pediatric COVID-19 hospital admissions. This sum is used as the
...
+| `confirmed_admissions_covid_1d_prop` and `confirmed_admissions_covid_1d_prop_7dav`                             | Sum of adult and pediatric confirmed COVID-19 hospital admissions occurring each day, per 100,000 population. <br/> **Earliest date available:** 2019-12-31                                                                          |
+| `sum_confirmed_suspected_admissions_covid_1d` and `sum_confirmed_suspected_admissions_covid_1d_7dav`           | Sum of adult and pediatric confirmed and suspected COVID-19 hospital admissions occurring each day. <br/> **Earliest date available:** 2019-12-31                                                                                    |
+| `sum_confirmed_suspected_admissions_covid_1d_prop` and `sum_confirmed_suspected_admissions_covid_1d_prop_7dav` | Sum of adult and pediatric confirmed and suspected COVID-19 hospital admissions occurring each day, per 100,000 population. <br/> **Earliest date available:** 2019-12-31                                                            |
```
Comment (krivard) [inline docs/api/covidcast-signals/hhs.md:40]: This fixes a whitespace problem in the rendered table
```suggestion
| `sum_confirmed_suspected_admissions_covid_1d_prop` and <br/> `sum_confirmed_suspected_admissions_covid_1d_prop_7dav` | Sum of adult and pediatric confirmed and suspected COVID-19 hospital admissions occurring each day, per 100,000 population. <br/> **Earliest date available:** 2019-12-31                                                            |
```

**Review by krivard — APPROVED** [review]
👍 🎉 Nice work!

**Review by krivard — APPROVED** [review]
silly robot dismissed my approving review 🤖

**Pull request merged**